Specifying Kafka version as 1.0.0 in Note for Kafka with EH (#3474)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md

| oidcClientID | N | Input/Output | The OAuth2 client ID that has been provisioned in the identity provider. Required when `authType` is set to `oidc` | `dapr-kafka` |
| oidcClientSecret | N | Input/Output | The OAuth2 client secret that has been provisioned in the identity provider: Required when `authType` is set to `oidc` | `"KeFg23!"` |
| oidcScopes | N | Input/Output | Comma-delimited list of OAuth2/OIDC scopes to request with the access token. Recommended when `authType` is set to `oidc`. Defaults to `"openid"` | `"openid,kafka-prod"` |
| version | N | Input/Output | Kafka cluster version. Defaults to 2.0.0. Please note that this needs to be mandatorily set to `1.0.0` for EventHubs with Kafka. | `1.0.0` |

#### Note

> Metadata `version` needs to be mandatorily set to `1.0.0` for anyone using EventHubs with Kafka.

Reviewer: ```suggestion
The metadata `version` must be set to `1.0.0` when using Azure EventHubs with Kafka.
```

Reviewer: Should this be `1.0.0.0` ?

Author: Actually, before `1.0.0`, versions had 4 parts, like `0.10.2.0` or `0.11.0.3`
After and from `1.0.0`, it has 3 parts.

Ref: https://kafka.apache.org/downloads#1.0.0
